src/systemd/cockpit-wsinstance-https@.service.in

```diff
@@ -2,12 +2,11 @@
 Description=Cockpit Web Service https instance %I
 Documentation=man:cockpit-ws(8)
 BindsTo=cockpit.service
-Requires=cockpit-session.socket cockpit-session-socket-user.service
-After=cockpit-session.socket cockpit-session-socket-user.service
+Requires=cockpit-session.socket
+After=cockpit-session.socket
 
 [Service]
 Slice=system-cockpithttps.slice
 ExecStart=@libexecdir@/cockpit-ws --for-tls-proxy --port=0
-User=cockpit-wsinstance-socket
-Group=cockpit-wsinstance-socket
-SupplementaryGroups=cockpit-session-socket
+DynamicUser=yes
```

Comment:
And here in the https-@ service you don't have an explicit `User=`. I would understand if you want to run http and https as different users, but TBH that doesn't buy us much -- with ptrace protection (which these processes already have even without our pending lockdown) this doesn't add much isolation, and the more important isolation is actually between *different* https@ instances (but we can't do that).

So for now I assume that the `User=` in http.service is a simple leftover, but I'd like to confirm from you.